lib/krane/kubernetes_resource/deployment.rb

```diff
@@ -155,6 +155,7 @@ def rollout_annotation_err_msg
 
     def deploy_failing_to_progress?
       return false unless progress_condition.present?
+      return false if deploy_started? && Time.now.utc - @deploy_started_at < progress_deadline
```

Comment:
Worth a comment here, since the deploy is indeed failing to progress, we're just ignoring that for now :)